.github/workflows/python-tests.yml

```
@@ -37,6 +37,9 @@ jobs:
           pip install --upgrade pip
           pip install .[testing]
 
+          apt install -y libsndfile1-dev
```

Indeed, got too used to the self-hosted machines

```suggestion
          sudo apt install -y libsndfile1-dev
```